src/Input/Checkbox/CheckboxStyle.ts

```diff
@@ -24,30 +44,20 @@ export const CheckboxContainer = styled.div<CheckboxProps>`
       content: '';
       display: block;
       position: absolute;
-      top: ${({ size }) => (size === 'small' ? '1.8px' : '3.3px')};
-      left: ${({ size }) => (size === 'small' ? '6.3px' : '7.5px')};
-      width: 6px;
-      height: 12px;
+      top: 0.13em;
```

Comment:
Hmm so this part is not affected by the size of the checkbox?

Reply:
What do you mean?

This part makes the checkbox's dimensions grow with the `font-size`:
```
width: 0.43em;
height: 0.86em;
```

This part positions the tick icon, relative to the top-left corner (which of course is not moving). It's a bit silly to do it this way imo, it would be better to use some alignment styles, but I can't change it to that easily:
```
top: 0.13em;
left: 0.45em;
```

Reply:
Ah no, I mean the `size` prop that can be small.
`top: ${({ size }) => (size === 'small' ? '1.8px' : '3.3px')};`

This part is removed, so what will happen to the Checkboxes with the size prop set as small?

Reply:
There's a line ` font-size: ${({ size }) => (size === 'small' ? '14px' : '16px')};` further up. The `size` determines the `font-size` which in turn determines everything else (that uses `em`).

Reply:
Oh cool had missed that line about the font-size.